modules/signatures/windows/office.py

```diff
@@ -132,23 +330,71 @@ def on_complete(self):
                 if "Sub Document_Open()" in macro["deobf"]:
                     return True
 
-class OfficeEpsStrings(Signature):
-    name = "office_eps_strings"
-    description = "Suspicious keywords embedded in an Encapsulated Post Script (EPS) file"
-    severity = 3
+class OfficeMacro(Signature):
+    name = "office_macro"
+    description = "Office document contains one or more macros"
+    severity = 2
     categories = ["office"]
-    authors = ["Cuckoo Technologies"]
+    authors = ["Kevin Ross"]
     minimum = "2.0"
 
-    keywords = [
-        "longjmp", "NtCreateEvent", "NtProtectVirtualMemory",
+    def on_complete(self):
+        office = self.get_results("static", {}).get("office", [])
+        for macro in office["macros"]:
```

Comment:
this fails every time when not dealing with office files

```
2017-09-04 13:29:31,340 [cuckoo.core.plugins] ERROR: Failed to run 'on_complete' of the office_macro signature
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/plugins.py", line 413, in call_signature
    if not signature.matched and handler(*args, **kwargs):
  File "/srv/cuckoo/cwd/signatures/windows/office.py", line 293, in on_complete
    for macro in office["macros"]:
TypeError: list indices must be integers, not str
```
maybe adding a check like this would make sense : 
```
        if not office:
            return self.has_marks()
```

Reply:
Checking for the presence of `macros` does the trick 
```        
if "macros" in office:
        for macro in office["macros"]:
            self.mark(
                macro_filename=macro["filename"],
                macro_stream=macro["stream"],
            )
        return self.has_marks()
```